docs/DeveloperGuide.md

```diff
@@ -241,6 +244,42 @@ _{more aspects and alternatives to be added}_
 _{Explain here how the data archiving feature will be implemented}_
 
 
+### <a name="adding_a_session"></a>Adding a session
+
+Adding a session to a class requires user input from the CLI, then the parser will parse the user input and 
+to obtain the name and the date of the session. The newly added session will also be filled with students' 
```

Comment:
Would "Adding a session to a class requires user input from the CLI. The parser will then parse the user input to obtain the name and the date of the session." flow better?

Reply:
I think I deleted some stuff by accident, will amend!